joern-cli/frontends/csharpsrc2cpg/src/main/scala/io/joern/csharpsrc2cpg/astcreation/AstCreator.scala

```diff
@@ -101,7 +101,6 @@ class AstCreator(
       case CatchDeclaration                                      => astForCatchDeclaration(nodeInfo)
       case PropertyDeclaration                                   => astForPropertyDeclaration(nodeInfo)
       case ExpressionElement                                     => astForExpressionElement(nodeInfo)
-      case _: LiteralExpr                                        => astForLiteralExpression(nodeInfo)
```

Comment:
Removed this, as the statement below would anyway pass control to the same method.